### PR TITLE
draft post for a solve of zdeveau's rare metal seq

### PR DESCRIPTION
**Pull request by BenGardiner**

### CODE REVIEW

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+# include something more complex.
+season: 19
+episode: 4
+layout: default
+title: NorthSec CTF 2021's Rare Metal Sequence
+# summary for social media cards. Ideal length 55 chars, max 200
+summary: Reverse a SNES crackme by Zack Deveau using radare2 and Jupyter Notebooks
+tags: reverse-engineering
+date_fr: 18 mai 2022
+date_long_fr: mercredi le 18 mai 2022
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:13]:
```suggestion
date_long_fr: mercredi le 18 mai 2022 18h à 21h
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+date_en: Mai 18th 2022
+date_long_en: Wednesday May 18th 2022
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:15]:
```suggestion
date_long_en: Wednesday May 18th 2022 18:00 to 21:00
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+description_fr: >
+   Les NSec 2021 CTF challenges "Rare metal sequencer" 1 et 2 etait des crackme par Zack Deveau. Zack a crée les challenge pour le Super Nintendo Entertainment System (SNES) et let participants devaient le reverser pour trouver les entrées du joypad qui vas le débloquer. Nous alons etudié comment utiliser radare2 pour le reverser en deux directions: en l'avant et en l'arrière. Aussi en prime nous allons montrer comment on peut fair le 2022 Nsec warmup challenge logically bonus flag.
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:17]:
```suggestion
   Le challenge du CTF NSec 2021 "Rare metal sequencer" 1 et 2 sont des crackme par Zack Deveau. Zack a créé les challenges pour le Super Nintendo Entertainment System (SNES) et les participants devaient faire de la rétro-ingénierie pour trouver les entrées du joypad pour résoudre le défi. Nous allons voir comment utiliser radare2 pour le *reverse* en deux directions: par l'avant et par l'arrière (forward and backward). En prime, nous allons montrer comment on peut résoudre le défi d'échauffement du CTF du NSec 2022 *logically bonus flag*.
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+description_en: >
+   Rare metal sequencer was a NSec 2021 CTF crackme by Zack Deveau. _This crackme_ runs on a Super Nintendo Entertainment System (SNES) and the required input is a sequence of SNES controller inputs. We will use radare2 to understand how the rare metal sequencers (levels 1 and 2) work. We will demonstrate the rerversing process with radare2 both forwards and backwards. As a bonus we will also do a bonus solve of the 2022 NSec warmup challenge logically bonus flag.
+tools_fr:
+  - Python
+  - Jupyter Notebooks
+  - radare2
+  - Connaissances en rétro-ingénierie un peut mais connaissances de SNES assembleur n'est pas nécessaire
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:24]:
```suggestion
  - Un peu de connaissances en rétro-ingénierie. Pas besoin de connaissance en assembleur SNES.
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+tools_en:
+  - Python
+  - Jupyter Notebooks
+  - radare2
+  - Knowledge in x86 reverse-engineering a little but knowledge of SNES assembly is not necessary
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:29]:
```suggestion
  - A little knowledge in x86 reverse-engineering. Knowledge of SNES assembly is **not** necessary
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+presented_by_fr: Défi présenté par Ben Gardiner ([@BenGardiner](https://github.com/BenGardiner)). Créé par Zack Deveau pour [NorthSec 2021](https://nsec.io/) ([@NorthSec_io](https://twitter.com/northsec_io)).
+presented_by_en: Presented by Ben Gardiner ([@BenGardiner](https://github.com/BenGardiner)). Made by Zack Deveau for [NorthSec 2021](https://nsec.io/) ([@NorthSec_io](https://twitter.com/northsec_io)).
+featured_img: /images/22-05_snes_zdeveau.png
+featured_img_alt: SNES Rare Metal Sequencer Screenshot
+challenge_assets:
+  - name: (SPOILERS) writeup
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:35]:
```suggestion
#  - name: (SPOILERS) writeup
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+    url: https://gist.github.com/BenGardiner/13fe76fd43f179a872acbb9f5729eb2c
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:36]:
```suggestion
#    url: https://gist.github.com/BenGardiner/13fe76fd43f179a872acbb9f5729eb2c
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+## Where
+
+* Follow the presentation [online on Twitch](https://twitch.tv/montrehack/)
+* Participate in teams or on your own [on Discord](https://discord.gg/4qfFwPX)
+
+## When
+
+:heavy_exclamation_mark: **New hours for the online formula!**
+
+{{ page.date_long_en | capitalize }} from **7pm to 10pm**
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:98]:
```suggestion
{{ page.date_long_en | capitalize }} from **6pm to 9pm**
```

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+## Où
+
+* Suivez la présentation [en ligne sur Twitch](https://twitch.tv/montrehack/)
+* Participez seul ou en équipe [sur Discord](https://discord.gg/4qfFwPX)
+
+## Quand
+
+:heavy_exclamation_mark: **Nouvelles heures pour la formule en ligne!**
+
+{{ page.date_long_fr | capitalize }} de **19h à 22h**
```
Comment (Res260) [inline _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:162]:
```suggestion
{{ page.date_long_fr | capitalize }} de **18h à 21h**
```

**Review by Res260 — COMMENTED** [review]
Thanks for the PR :)

File: _posts/2022-05-18-solving-zdeveau-21-with-radare2.md (hunk 3 above)
Comment (BenGardiner) [reply _posts/2022-05-18-solving-zdeveau-21-with-radare2.md:17]: thank you for fixing my attempt at the french description!

**Comment by BenGardiner** [comment]
Thanks @Res260 / it's my pleasure :) All edits accepted.

**Pull request merged**